backend/services/operations_test.go

```diff
@@ -238,12 +238,30 @@ func TestReadOperation(t *testing.T) {
 		require.Equal(t, masterOp.Slug, retrievedOp.Slug)
 		require.Equal(t, masterOp.Name, retrievedOp.Name)
 		require.Equal(t, masterOp.Status, retrievedOp.Status)
+		require.Equal(t, 6, retrievedOp.NumUsers)
+		require.Equal(t, true, retrievedOp.Favorite)
+		require.Equal(t, 8, retrievedOp.NumEvidence)
+		require.Equal(t, 12, retrievedOp.NumTags)
+		require.Equal(t, 1, len(retrievedOp.TopContribs))
+		require.Equal(t, "harry.potter", retrievedOp.TopContribs[0].Slug)
+		require.Equal(t, int64(2), retrievedOp.EvidenceCount.CodeblockCount)
+		require.Equal(t, int64(6), retrievedOp.EvidenceCount.ImageCount)
+		require.Equal(t, int64(0), retrievedOp.EvidenceCount.RecordingCount)
+		require.Equal(t, int64(0), retrievedOp.EvidenceCount.EventCount)
+		require.Equal(t, int64(0), retrievedOp.EvidenceCount.HarCount)
```

Comment:
These all seem like they might break if the test data changes. I don't have an immediate solution, but something to keep in mind.

Reply:
yeah that's a fair point - my thought was that the test data probably won't change soon, and even if it does, it will be fairly easy to change this test. I could have copied over some functionality to return the same response (like I did with getting all the operations), but it would have been more work to setup, and would be brittle; not if the test data changes, but if my functionality involved with getting an operation changes